### PR TITLE
DOC: v7.4.0

### PR DESCRIPTION
**Pull request by ZLLentz**
## Description

- Run the release notes script to accumulate @spenc333 's docs
- Recategorize all under `Device Updates` and add more context/docs links
- Swap to github usernames for proper release credit
 
I think Robert and I were included because we offered a bunch of reviewer suggestions, but this release is really all @spenc333  

## Motivation and Context

- `pcds_conda` tag this week
- Good, useful progress in `pcdsdevices` to deploy

## How Has This Been Tested?

I'll check the generated docs page to make sure the build works and that the links I added work as intended on click

## Where Has This Been Documented?

This is docs

## Pre-merge checklist
- [x] Code works interactively
- [x] Code contains descriptive docstrings, including context and API
- [x] New/changed functions and methods are covered in the test suite where possible
- [ ] Test suite passes locally
- [x] Test suite passes on GitHub Actions
- [x] Ran docs/pre-release-notes.sh and created a pre-release documentation page
- [x] Pre-release docs include context, functional descriptions, and contributors as appropriate

### CODE REVIEW

File: docs/source/releases.rst [hunk 1]
```diff
@@ -1,6 +1,32 @@
 Release History
 ###############
 
+
+v7.4.0 (2023-05-08)
+===================
+
+Device Updates
+--------------
+- `IMS` class:
```
Comment (klauer) [inline docs/source/releases.rst:10]: Not sure if sphinx will render it differently, but GitHub rich view shows this:
image

Comment (ZLLentz) [reply docs/source/releases.rst:10]: Sphinx did render it as GitHub did, I'm going to need to adjust it.
I also want to tweak the docstring of `IMS.diff_configuration` to reflect the new return type.

**Comment by ZLLentz** [comment]
The links from the docs page to e.g. the IMS page did work in the build coming off of the first CI run here

**Review by klauer — APPROVED** [review]

**Comment by ZLLentz** [comment]
The rebuild also looks good!
I will merge this as-is so we can proceed with the pcds_envs tomorrow
It is never too late to revise the release notes if there is an error or an omission

**Pull request merged**